ConsoleApplication/app.config

```diff
@@ -1,13 +1,12 @@
 <?xml version="1.0"?>
 <configuration>
   <appSettings>
-     <!--Windows Azure AD tenant domain name--> 
-    <add key="TenantDomainName" value="To be filled in"/>
      <!--Service principal credentials-->  
-    <add key="AppPrincipalId" value="edd4904c-ac06-4f13-b8b5-9ec799e82bd3"/>
+    <add key="AppPrincipalId" value="To be filled in"/>
```

Comment:
Move this to a separate .config file that is not checked in to the repo (added to .gitignore). This is to prevent inadvertently leaking your app ID.

Look at https://github.com/microsoftgraph/msgraph-training-aspnetmvcapp/tree/master/Demos/03-add-msgraph/graph-tutorial. In that sample, a separate PrivateSettings.config is used to hold OAuth settings, and is merged into Web.config by using this line:

```
<appSettings file="PrivateSettings.config">
```

Then, a copy of PrivateSettings.config, named [PrivateSettings.config.example](https://github.com/microsoftgraph/msgraph-training-aspnetmvcapp/blob/master/Demos/03-add-msgraph/graph-tutorial/PrivateSettings.config.example) is added to the repo with placeholders, so devs can just rename the file and plug in their own app ID/secret.

Reply:
Thanks for this @jasonjoh,
I have made updates to this where the Readme has been updated to instruct the user to first rename the *appsettings.json.example* file and the *appsettings.json* been added to the .gitignore to prevent leaking of ones appID.